spec/index.md

```diff
@@ -2081,6 +2081,10 @@ A `get_utxos_request` without the optional `filter` results in a request that co
 
 The recommended workflow is to issue a request with the desired number of confirmations. If the `next_page` field in the response is not empty, there are more UTXOs than in the returned vector. In that case, the `page` field should be set to the `next_page` bytes in the subsequent request to obtain the next batch of UTXOs.
 
+### IC method `bitcoin_get_utxos_untrusted` {#ic-bitcoin_get_utxos_untrusted}
+
+This method is identical to [`bitcoin_get_utxos`](#ic-bitcoin_get_utxos). The only difference is that it is exposed as an uncertified query call, and its results can therefore not be trusted.
```

Comment:
I would prefer to perhaps call the functions `bitcoin_get_utxos_query` or so, and tone this text down a little bit to say something like "as it's a query, the answer comes from a single replica and is therefore not appropriate for security-sensitive applications" or so.

Reply:
Done.